### ISSUE TITLE
magic_number flags well-known constants (HTTP codes, time units)

### ISSUE THREAD

**Issue by rsionnach** [rsionnach/sloppylint#7]
## Issue
The `magic_number` pattern flags universally understood constants like HTTP status codes and time units.

## Examples
```python
# HTTP status codes - universally understood
if response.status == 200:  # FLAGGED
if response.status == 404:  # FLAGGED
if response.status == 500:  # FLAGGED

# Time constants - self-documenting when assigned
SECONDS_PER_MINUTE = 60  # FLAGGED
MINUTES_PER_HOUR = 60    # FLAGGED
HOURS_PER_DAY = 24       # FLAGGED
```

## Expected Behavior
Well-known constants should not be flagged:
- HTTP status codes (200, 201, 204, 301, 302, 400, 401, 403, 404, 500, 502, 503)
- Time units (60 for seconds/minutes, 24 for hours, 365 for days)
- Common computing values (256, 1024, 8080, etc.)

## Suggested Fix
Add a whitelist of well-known magic numbers that don't need named constants.

**Comment by rsionnach** [comment]
This was already addressed. The `magic_number` pattern has a `WELL_KNOWN_NUMBERS` whitelist that includes:

- **HTTP status codes**: 200, 201, 204, 301, 302, 400, 401, 403, 404, 500, 502, 503, etc.
- **Time units**: 24, 60, 365, 366
- **Powers of 2**: 256, 512, 1024, 2048, 4096, etc.
- **Common ports**: 80, 443, 8080
- **Common defaults**: 10, 50, 99, 128, 255

These numbers are not flagged as magic numbers. Closing as fixed.
